config/manager/manager.yaml

```diff
@@ -43,17 +43,19 @@ spec:
       containers:
       - command:
         - /root/manager
-        # args:
+        args:
+          - "--auditInterval=30"
           # - "--alsologtostderr"
           # - "--stderrthreshold=INFO"
           # - "-v=100"
-        image: controller:latest
+        image: quay.io/open-policy-agent/gatekeeper:v3.0.4-alpha.0
         imagePullPolicy: Always
         name: manager
```

Comment:
same: Should we also change `manager` to `gatekeeper-manager` like the clusterrole and clusterrolebinding?

Reply:
Seems redundant given it's in the gatekeeper-controller-manager pod